packages/webview_flutter/webview_flutter_wkwebview/lib/src/ui_kit/ui_kit_api_impls.dart

```diff
@@ -0,0 +1,74 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'dart:async';
+import 'dart:math';
+
+import 'package:flutter/services.dart';
+
+import '../common/instance_manager.dart';
+import '../common/web_kit.pigeon.dart';
+import '../web_kit/web_kit.dart';
+import 'ui_kit.dart';
+
+/// Host api implementation for [UIScrollView].
+class UIScrollViewHostApiImpl extends UIScrollViewHostApi {
+  /// Constructs a [UIScrollViewHostApiImpl].
+  UIScrollViewHostApiImpl({
+    BinaryMessenger? binaryMessenger,
+    InstanceManager? instanceManager,
+  })  : instanceManager = instanceManager ?? InstanceManager.instance,
+        super(binaryMessenger: binaryMessenger);
+
+  /// Maintains instances stored to communicate with Objective-C objects.
+  final InstanceManager instanceManager;
+
+  /// Converts objects to instances ids for [createFromWebViewFromInstance].
```

Comment:
I'm not sure what this means exactly; it doesn't return anything, so what is being converted exactly?

I'm also confused since it says that it's "for [createFromWebViewFromInstance]", but that's the function being documented.